### PR TITLE
javadocs: fix code examples

### PR DESCRIPTION
**Pull request by char16t**
**Problem:**
Javadocs comments at Tracer interface contains call of unexistent method OpenTelemetry.getTracer()

**Solution:**
Replace OpenTelemetry.getTracer() with OpenTelemetry.getTracer(String, String) in examples at javadocs

### CODE REVIEW

File: api/all/src/main/java/io/opentelemetry/api/trace/Tracer.java [hunk 1]
```diff
@@ -22,7 +22,7 @@
  *
  * <pre>{@code
  * class MyClass {
- *   private static final Tracer tracer = OpenTelemetry.getTracer();
+ *   private static final Tracer tracer = OpenTelemetry.getTracer("instrumentation-library-name", "1.0.0");
```
Comment (jkwatson) [inline api/all/src/main/java/io/opentelemetry/api/trace/Tracer.java:25]: This is also not a currently existing API. We'd need to change this to either `GlobalOpenTelemetry` or (preferable) a lower-case `openTelemetry` instance.

**Review by jkwatson — APPROVED** [review]
Thanks!

**Comment by jkwatson** [comment]
@char16t It looks like you need to run the formatter on this change. 

`./gradlew spotlessApply`

Thanks

**Comment by char16t** [comment]
Pipeline broken due to Checkstyle violation. I'll fix it

```
Error: Execution failed for task ':api:all:checkstyleMain'.
> Checkstyle rule violations were found. See the report at: file:///home/runner/work/opentelemetry-java/opentelemetry-java/api/all/build/reports/checkstyle/main.html
  Checkstyle files with violations: 1
  Checkstyle violations by severity: [error:2]
Error: [Task :api:all:checkstyleMain] [LineLength] Line is longer than 100 characters (found 107).
Error: [Task :api:all:checkstyleMain] [LineLength] Line is longer than 100 characters (found 107).
Warning: Gradle Build Scan: https://gradle.com/s/q3tbkjb64emwa

```

**Comment by char16t** [comment]
The formatter did not apply to the long lines in the comment. Fixed manually.

**Pull request merged**